### PR TITLE
fix docs on how to run tests

### PR DESCRIPTION
**Pull request by hxpaul**
#### What does this PR do? (please provide any background)
updates documentation on how to run tests, as it is broken.
Also removes ref to yeoman generator as it is out of date.
#### What tests does this PR have?
none
#### How can this be tested?
read the readme, then `npm run docs`
#### What gif best describes how you feel about this work?
---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked all the tests are passing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
 
 Adding a new component
 ---
-#### Install our [yeoman component generator](https://github.com/holidayextras/generator-ui-component)
```
Comment (robhuzzey) [inline README.md]: Why are we removing this bit? I assume we are assuming everybody has it installed?

File: README.md [hunk 2]
```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
...
-```bash
-npm install -g holidayextras/generator-ui-component
-```
-
 #### Creating the component
 Generate a component & follow the prompts
```
Comment (robhuzzey) [inline README.md]: My comment leads onto this point where we say generate a component... how?

File: README.md (hunk 1 above)
Comment (hxpaul) [inline README.md]: we should not be using this generator should we? It does not create anything like what we would recommend now does it?

File: README.md (hunk 2 above)
Comment (hxpaul) [inline README.md]: we will need some more docs in here soon

**Comment by jodiedoubleday** [comment]
Looks good :shipit:
Sooo happy we are removing the generator!

**Comment by cameronviner** [comment]
Taking DEV

File: README.md [hunk 3]
```diff
@@ -59,18 +59,13 @@ solutions. If you cannot find your specific problem then please create a
...
 ```bash
 cd src/components
 mkdir your-new-component
 cd your-new-component
-yo ui-component:external
+~~yo ui-component:external~~
```
Comment (cameronviner) [inline README.md]: Should we just remove this line if we are removing the link to install the generator? There are also references to it in https://github.com/holidayextras/ui-toolkit/blob/master/CONTRIBUTING.md should these also be removed?

**Comment by jodiedoubleday** [comment]
I have a feeling that we will introduce the generator again, but better.

**Comment by cameronviner** [comment]
Looks good to me :+1:

**Pull request merged**